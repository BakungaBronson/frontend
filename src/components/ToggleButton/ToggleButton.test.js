import React from 'react';

import { shallow } from 'enzyme';

import ToggleButton from './index';

describe('Toggle Button Component', () => {
  it('Renders a Toggle button component', () => {

    // const buttonEvent = jest.fn();
    const ButtonComponent = shallow(<ToggleButton />);
    
    // expect(ButtonComponent.find('.ToggleButtonContainer').hasClass('ToggleButtonContainer')).toBe(true);
    // expect(component.find('#item-id').prop('style')).toHaveProperty('backgroundColor' 'black');
    expect(ButtonComponent).toMatchSnapshot();

  });
});