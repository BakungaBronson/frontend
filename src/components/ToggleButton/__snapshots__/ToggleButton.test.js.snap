// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggle Button Component Renders a Toggle button component 1`] = `ShallowWrapper {}`;
